build/repo.props

```
@@ -18,7 +18,7 @@
     <SignCheckExclusionsFile>$(RepositoryRoot)eng\signcheck.exclusions.txt</SignCheckExclusionsFile>
     <SharedSourcesFolder>$(RepositoryRoot)src\Shared\</SharedSourcesFolder>
     <SharedFxArchitecture Condition="'$(SharedFxArchitecture)' == ''">$(SharedFxRid.Substring($([MSBuild]::Add($(SharedFxRid.LastIndexOf('-')), 1))))</SharedFxArchitecture>
-    <BuildSiteExtensions>false</BuildSiteExtensions>
+    <BuildSiteExtensions>true</BuildSiteExtensions>
```

```suggestion
    <BuildSiteExtensions Condition=" '$(VersionPrefix)' == '2.2.5' " >true</BuildSiteExtensions>
```

Instead of us having to remember to revert this later, can you make this conditional?